ibm_catalog.json

```diff
@@ -0,0 +1,114 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-icd-redis",
+      "label": "ICD Redis module",
```

Comment:
I missed this. Need to refer to this as `IBM Cloud Databases for Redis module` or `Databases for Redis module`